packages/renown/package.json

```diff
@@ -37,9 +37,6 @@
     "clean": "rimraf dist/",
     "clean:node_modules": "rimraf node_modules"
   },
-  "peerDependencies": {
-    "document-model": "workspace:*"
```

Comment:
Renown is also a ts-only package, it only imports the `User` type from "document-model". It probably needs to change the bundle to include the type definition.